src/ai/backend/manager/models/group.py

```diff
@@ -378,7 +378,8 @@ async def batch_load_by_user(
         cls,
         graph_ctx: GraphQueryContext,
         user_ids: Sequence[uuid.UUID],
-        type: list[ProjectType] = [ProjectType.GENERAL],
+        *,
+        type: list[ProjectType] | None = None,
```

Comment:
Is there a specific reason you decided to expose every types of project by default here? When forming the model store sprint (which is directly related to this `type` parameter) we decided to reveal such projects only if the type was explicitly specified by requester.